pages/common/git-subtree.md

```diff
@@ -5,19 +5,19 @@
 
 - Add a git repository as a subtree:
 
-`git subtree add --prefix={{path/to/directory/}} --squash {{repository_url}} {{master}}`
+`git subtree add --prefix={{path/to/directory/}} --squash {{repository_url}} {{branch_name}}`
 
 - Update subtree repository to its latest commit:
 
-`git subtree pull --prefix={{path/to/directory/}} {{repository_url}} {{master}}`
+`git subtree pull --prefix={{path/to/directory/}} {{repository_url}} {{branch_name}}`
 
-- Merge a subtree repository into master:
+- Merge a subtree repository into a specified branch:
```

Comment:
I'm not sure if the last parameter refers to the source or the target branch. As the [author of this page](https://github.com/tldr-pages/tldr/pull/4012), could you comment, @einverne?

Reply:
Looking at the man page, it details this:

> Merge recent changes up to <commit> into the <prefix> subtree.

So I think that it's actually going in the opposite direction to that of the description here.

Again, I haven't actually used the command before, so I'm not sure. Let's wait for @einverne's response.

Reply:
According to the document, it says:

> git subtree merge -P <prefix> <commit>
> Merge recent changes up to <commit> into the <prefix> subtree.

What the `subtree merge` do is to merge the latest commit to the subtree.

Reply:
Ohhh I see. Thanks for the insight here, @einverne! So could we reword to `{{subtree_branch_name}}` here then perhaps?

Reply:
@sbrl Yes, and the origin description may not be accurate. How about

> Merge recent changes up to {{latest_subtree_commit}} into the subtree.

Reply:
Applied suggestion, @einverne.

Is this better?